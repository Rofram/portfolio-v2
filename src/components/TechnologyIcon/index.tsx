import * as S from './styles'

const TechnologyIcon = () => (
  <S.Wrapper>
    <h1>TechnologyIcon</h1>
  </S.Wrapper>
)

export default TechnologyIcon
