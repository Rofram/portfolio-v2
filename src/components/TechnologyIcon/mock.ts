import { TechnologyIconProps } from '.'

const mock: TechnologyIconProps = {
  src: '/img/techs/react.svg',
  alt: 'React Icon'
}

export default mock
