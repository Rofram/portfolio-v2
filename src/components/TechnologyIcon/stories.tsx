import { Story, Meta } from '@storybook/react'
import TechnologyIcon from '.'

export default {
  title: 'TechnologyIcon',
  component: TechnologyIcon
} as Meta

export const Default: Story = () => <TechnologyIcon />
