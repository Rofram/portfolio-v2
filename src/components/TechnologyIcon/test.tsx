import { render, screen } from '@testing-library/react'

import TechnologyIcon from '.'

describe('<TechnologyIcon />', () => {
  it('should render the heading', () => {
    const { container } = render(<TechnologyIcon />)

    expect(
      screen.getByRole('heading', { name: /TechnologyIcon/i })
    ).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
