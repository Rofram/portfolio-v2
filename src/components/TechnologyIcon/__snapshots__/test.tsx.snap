// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TechnologyIcon /> should render the heading 1`] = `
<main
  class=""
>
  <h1>
    TechnologyIcon
  </h1>
</main>
`
